editoast/src/views/study.rs

```diff
@@ -223,15 +257,13 @@ async fn list(
     )
 )]
 #[get("")]
-async fn get(db_pool: Data<DbPool>, path: Path<(i64, i64)>) -> Result<Json<StudyWithScenarios>> {
+async fn get(db_pool: Data<DbPool>, path: Path<(i64, i64)>) -> Result<Json<StudyResponse>> {
     let (project_id, study_id) = path.into_inner();
 
     // Check if project exists
-    if Project::retrieve(db_pool.clone(), project_id)
-        .await?
-        .is_none()
-    {
-        return Err(ProjectError::NotFound { project_id }.into());
+    let project = match Project::retrieve(db_pool.clone(), project_id).await? {
+        None => return Err(ProjectError::NotFound { project_id }.into()),
+        Some(project) => project,
     };
```

Comment:
Rather recently introduced syntax shortcut.

```suggestion
    let Some(project) = Project::retrieve(db_pool.clone(), project_id).await? else {
        return Err(ProjectError::NotFound { project_id }.into());
    };
```